src/dashboard/trulens/dashboard/pages/Evaluations.py

```diff
@@ -149,24 +149,44 @@ def extract_metadata(row: pd.Series) -> str:
     return str(record_data["meta"])
 
 
-apps = list(app["app_id"] for app in lms.get_apps())
+app_ids = list(app["app_id"] for app in lms.get_apps())
+
+
+@st.cache_data
```

Comment:
if we cache apps, new apps won't be reflected in the dashboard without manually reloading

Reply:
addressed all